package-lock.json

```diff
@@ -1,922 +0,0 @@
-{
```

Comment:
I'm not sure why did you remove this file?

Reply:
My mistake, I use yarn ;)